src/components/NcListItem/NcListItem.vue

```diff
@@ -381,7 +381,7 @@
 					<!-- Main content -->
 					<div class="list-item-content">
 						<div class="list-item-content__main">
-							<div class="list-item-content__name">
+							<div class="list-item-content__name" :class="{ 'limit-width': oneLine }">
```

Comment:
1. Please use BEM (so it would be `list-item-content__name--one-line`)
2. But this is not needed as the modifier is already set on the parent class  (see comment below)